management/web_update.py

```diff
@@ -82,6 +82,7 @@ def read_conf(conf_fn):
 
 	# Build an nginx configuration file.
 	nginx_conf = read_conf("nginx-top.conf")
+	nginx_conf = re.sub("{{phpver}}", get_php_version(), nginx_conf)
```

Comment:
I don't understand, why is this change needed? Maybe add another inline comment here?

Reply:
The nginx configuration file is a template, where the php version is hardcoded. This line fills in the php version in the nginx conf file. I can add a comment.

Reply:
Yes please add, thanks